fedimint-api/src/module/mod.rs

```diff
@@ -440,6 +440,68 @@ where
     }
 }
 
+pub enum ConsensusProposal<CI> {
+    /// Trigger new epoch immediately including these consensus items
+    Force(Vec<CI>),
+    /// Do not trigger new epoch, but if other module does, use these
+    /// consensus items.
+    // TODO: turn into `(Vec<CI>, Update)` where `Updates` is a future
+    // that will return a new `ConsensusProposal` when updates are available.
+    // This wake we can get rid of `await_consensus_proposal`
+    Lazy(Vec<CI>),
+}
+
+impl<CI> From<Vec<CI>> for ConsensusProposal<CI> {
```

Comment:
Wonder if it is safer to force modules to implement this themselves to avoid accidentally always triggering epochs.

Reply:
I don't like hiding this kind of logic in a `From`/`into`. Should be more explicit like `new_auto_lazy(…)`

Reply:
You're right. It did feel fishy. Done.